webapp/channels/src/components/admin_console/admin_definition.tsx

```diff
@@ -1130,6 +1130,191 @@ const AdminDefinition: AdminDefinitionType = {
                     ],
                 },
             },
+            export_storage: {
+                url: 'environment/export_storage',
+                title: defineMessage({id: 'admin.sidebar.exportStorage', defaultMessage: 'Export Storage'}),
+                isHidden: it.any(
+                    it.not(it.licensedForFeature('Cloud')),
+                    it.not(it.licensedForSku(LicenseSkus.Enterprise)),
+                    it.configIsFalse('FeatureFlags', 'CloudDedicatedExportUI'),
+                ),
+                schema: {
+                    id: 'ExportFileSettings',
+                    name: defineMessage({id: 'admin.sidebar.exportStorage', defaultMessage: 'Export Storage'}),
+                    settings: [
+                        {
+                            type: 'bool',
+                            key: 'FileSettings.DedicatedExportStore',
+                            label: defineMessage({id: 'admin.exportStorage.dedicatedExportStore', defaultMessage: 'Enable Dedicated Export Store:'}),
+                            help_text: defineMessage({id: 'admin.exportStorage.dedicatedExportStoreDescription', defaultMessage: 'When enabled, Mattermost will use a dedicated export storage bucket for all export operations. This is required for Mattermost Cloud deployments.'}),
```

Comment:
If the setting is disabled, what file store is used for exporting? I suppose it's the standard file storage, but where does that point to?

Reply:
Correct it should use the standard file storage